src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -253,7 +274,15 @@ function MoneyRequestPreview(props) {
                         </View>
                         <View style={[styles.flexRow]}>
                             <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter]}>
-                                <Text style={styles.textHeadline}>{getDisplayAmountText()}</Text>
+                                <Text
+                                    style={[
+                                        styles.moneyRequestPreviewAmount,
+                                        StyleUtils.getFontSizeStyle(variables.fontSizeXLarge - getFontSizeToSubstract()),
+                                    ]}
+                                    maxNumberOfLines={3}
```

Comment:
`maxNumberOfLines={3}` is a fallback for Android & iOS which would make the amount visible if it would not fit in 1 row. I've tested all resolutions as seen in the **web** video and this should not happen, but offers a fallback if something goes wrong.

Reply:
Lets say we didn't have this fallback, the worst thing that would happen is that the amount overlaps with the avatars again, right? 

I think we should drop the fallback and just fix that if/when it occurs. Else someone might think there is a purpose to the 3 lines prop.

Reply:
@Julesssss removed `maxNumberOfLines={3}` in last commit

Reply:
@Julesssss I've actually added it back as `numberOfLines={1}`. This way, the amount would never overlap the avatars again, but instead would show like this:  

<img src="https://github.com/Expensify/App/assets/56603839/719ed47d-d7ae-4ac8-a4d1-2632f366c79d" width="400px">  
  
  
> Else someone might think there is a purpose to the 3 lines prop.

Now it has a purpose as our goal is to show the full amount on 1 line, but if we fail under some conditions, we'll have the option to adjust values in the helper method, but would at least not overlap the avatars.

I'll remove it again if you think it's a bad idea, let me know!

Reply:
I see we are currently using `numberOfLines={1}` in many places in the app as of now, but I'm ok with removing it. So, should I?

Reply:
This seems to remove the **Desktop** app problem in the **350-360px** interval:  

<img src="https://github.com/Expensify/App/assets/56603839/8730e148-2975-4032-82c6-80705f9e81b0" witdh="250px">

And would also permit narrower than **320px** devices to work as they would not overlap the avatar icons. I'm inclined towards keeping it, but I'll do as you say!